src/ipa_pc/mod.rs

```diff
@@ -330,6 +339,8 @@ impl<G: AffineCurve, D: Digest> InnerProductArgPC<G, D> {
             round_challenge = fs_rng.squeeze_128_bits_challenge();
 
             round_challenges.push(round_challenge);
+
+            // round_challenge is guaranteed to be non-zero by squeeze function
```

Comment:
I'd rather state "round_challenge as produced by the squeeze function is overwhelmingly non-zero"